tests/integration/telemetry/common/wasm/maint_test.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"istio.io/istio/pkg/test/framework/components/registryredirector"
 	"istio.io/istio/pkg/test/framework/resource"
-	common "istio.io/istio/tests/integration/telemetry/stats/prometheus"
+	"istio.io/istio/tests/integration/telemetry/common"
```

Comment:
is the file name starting with `maint` intentional?  should it be `main` or does the `t` signify anything?